blogs/scripts/lib-franklin.js

```diff
@@ -558,6 +558,110 @@ export function loadHeader(header) {
   return loadBlock(headerBlock);
 }
 
+/**
```

Comment:
The `lib-franklin.js` is more of a platform level script that should not be changed except for very unique circumstances.
Could this have been done by modifying the `header` block?

Reply:
Agree with rob, this should ideally be in the header block, but at the very least, be in scripts.js